app/templates/components/dialogs/prioritize-build-modal.hbs

```diff
@@ -0,0 +1,66 @@
+<Modal
+  class="repo-actions-modal"
+  @isVisible={{@isOpen}}
```

Comment:
As long as there's a local `isOpen` property, defined on the component class, should it be used here instead of just reading attribute directly? E.g. `@isVisible={{this.isOpen}}`. Currently, it'll ignore the property on the component class.

Reply:
Used {{this.isOpen}} instead of {{@isOpen}} as suggested.